src/commands/internal-auth.json

```diff
@@ -0,0 +1,31 @@
+{
+    "INTERNALAUTH": {
+        "summary": "Initiates an internal connection.",
+        "complexity": "O(1)",
+        "group": "connection",
+        "since": "8.0.0",
+        "arity": 2,
+        "function": "internalAuthCommand",
+        "command_flags": [
+            "NOSCRIPT",
+            "LOADING",
+            "STALE",
+            "FAST",
+            "NO_AUTH",
+            "SENTINEL",
```

Comment:
Not sure what SENTINEL flag means but I am pretty sure we do not need it here.

Reply:
It means that the command exists in sentinel mode. Not sure we need it, but took inspiration from the `auth.json` command definition.

Reply:
No need for this command on sentinel.